app/views/donations/_edit.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class="modal fade" id="donationeditModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+  <div class="modal-dialog modal-sm" role="document">
+    <div class="modal-content">
+      <div class="modal-body">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+
+        <%= form_for(@donation) do |f| %>
+        <div class="form-group">
+          <%= f.label :amount, class: "control-label" %>
+          <%= f.text_field :amount, class: "form-control" %>
+        </div>
+
+        <div class="form-group">
+          <label for="causeOrEvent" class="form-control-label">Cause:</label>
+          <%= f.select :cause_id, options_for_select(Cause.all.map{|e| [e.name, e.id]}), {}, class:"form-control" %>
+        </div>
+
+        <div class="form-group">
+          <label for="causeOrEvent" class="form-control-label">Event:</label>
+          <%= f.select :event_id, options_for_select(Event.all.map{|e| [e.name, e.id]}), {}, class:"form-control" %>
```

Comment:
Instead of `#map`, use `#pluck`. It will just get the values from the database without having to load the models:

```
options_for_select(Event.pluck(:name, :id))
```

Same for `Cause` above. 😀